### PR TITLE
new package: py-lcls-krtc

### PR DESCRIPTION
**Pull request by eugeneswalker**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-lcls-krtc/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    """Very small utility class for using Kerberos authentication with Python requests."""
+
+    pypi = "lcls-krtc/lcls-krtc-0.2.0.tar.gz"
+
+    maintainers = ["valmar"]
+
+    version("0.2.0", sha256="20e6327d488d23e29135be44504bf7df72e4425a518f4222841efcd2cd2985f9")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-kerberos", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-lcls-krtc/package.py]:
```suggestion
    depends_on("py-pykerberos", type=("build", "run"))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-lcls-krtc/package.py (hunk 1 above)
Comment (eugeneswalker) [reply var/spack/repos/builtin/packages/py-lcls-krtc/package.py]: Are you sure?

Here using `spack@develop` (c85877566f4793ae97848d9c23828d9fe8dd7edd from `Fri Apr 28 15:24:24 2023 +0200`)
```
(apc351) eugenewalker@mac work % spack info py-pykerberos
==> Error: Package 'py-pykerberos' not found.
You may need to run 'spack clean -m'.
```

I did do a test build of `py-lcls-krtc`, as proposed in this PR, and it worked.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-lcls-krtc/package.py]: Yes, you'll need to add a package for it. Both pykerberos and kerberos exist on PyPI, and it seems like they are different packages maintained by different people. The installation process doesn't actually check that the package runs correctly.

Comment (eugeneswalker) [reply var/spack/repos/builtin/packages/py-lcls-krtc/package.py]: OK. Thanks for pointing this out. I will create a PR to add `py-pykerberos` and update this PR accordingly. Let us defer merge until then.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-lcls-krtc/package.py]: You can always add it to this PR if that's quicker

**Comment by valmar** [comment]
This looks OK to me. Just to be clear, this package is supposed to correspond to this: `https://github.com/pcdshub/krtc` , which lists pykerberos as a dependency. It therefore looks like `py-pykerberos` is the right dependency

**Comment by eugeneswalker** [comment]
> It therefore looks like `py-pykerberos` is the right dependency

I am creating a Spack package for `py-pykerberos`, will add it to this PR, and also update the dependency so that it is for that.

**Comment by eugeneswalker** [comment]
Just updated this PR to include:
* new package: `py-pykerberos` 
* `py-lcls-krtc`now depends on `py-pykerberos` (NOT `py-kerberos`)

**Review by adamjstewart — APPROVED** [review]

**Comment by eugeneswalker** [comment]
@spackbot run pipeline

**Pull request merged**